### ISSUE TITLE
typedef-whitespace call-signature=nospace creates a lint error for multiline functions

### ISSUE THREAD

**Issue by Josh211ua** [palantir/tslint#787]
Setting the typedef-whitespace call-signature rule to equal nospace leads to a lint failure in the following code:

```ts
function a(
  param: any
): any {
}
```

I believe this should not lead to a lint error as there is no space between the closing paren and the colon.
It seems the lint rule is getting confused by the whitespace that precedes the paren because this code has no lint error:
```ts
function a(
  param: any): any {
}
```

**Comment by b0r3as** [comment]
Could this also be fixed with #958?

**Comment by adidahiya** [comment]
fixed and verified in #958
